19_Most Consective Repetitions.py

```diff
@@ -1,14 +1,35 @@
 # Question 8: Longest Repetition
 
-# Define a procedure, longest_repetition, that takes as input a 
-# list, and returns the element in the list that has the most 
-# consecutive repetitions. If there are multiple elements that 
-# have the same number of longest repetitions, the result should 
-# be the one that appears first. If the input list is empty, 
+# Define a procedure, longest_repetition, that takes as input a
+# list, and returns the element in the list that has the most
+# consecutive repetitions. If there are multiple elements that
+# have the same number of longest repetitions, the result should
+# be the one that appears first. If the input list is empty,
 # it should return None.
 
 def longest_repetition(li):
-    # Your code here        
+    if not li:
+        return None
+
+    element=''
+    repetitions=0
+
+    i=0
+    while i<len(li)-1:

```

Comment:
Your code is working very well. It would be interesting if you can also solve it without indexing the list. Just by using `for element in li`.